applications/apps.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class ApplicationsConfig(AppConfig):
+    name = 'applications'
+
+    def ready(self):
+        super(ApplicationsConfig, self).ready()
+        from .signals import clean_draftapplication
+        clean_draftapplication
```

Comment:
see below about `clean_draft_application` naming
```suggestion
        clean_draft_application
```